packages/bento-design-system/src/util/defaultConfigs.tsx

```diff
@@ -204,31 +204,30 @@ export const disclosureGroup: DisclosureGroupConfig = {
 };
 
 export const feedback: FeedbackConfig = {
-  background: null,
-  positiveIllustration: IllustrationPositive,
-  negativeIllustration: IllustrationNegative,
+  positiveIcon: IconPositiveCircle,
+  negativeIcon: IconNegative,
   title: {
     medium: { kind: "title", size: "large" },
-    large: { kind: "display", size: "small" },
+    large: { kind: "headline", size: "medium" },
```

Comment:
Let's document the changes in this file in the migration guide: if the user wants to preserve the same UI, they need to manually set the config to what it was before.
I would provide a code snippet to add to their config.